docs/misconfiguration/policy/builtin.md

```diff
@@ -0,0 +1,23 @@
+# Built-in Policies
+
+## Policy Sources
+
+Built-in policies are mainly written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/).
+Those policies are managed under [AppShield repository](https://github.com/aquasecurity/appshield).
+Only Terraform's policies are currently powered by [tfsec](https://github.com/tfsec/tfsec).
+
+| Config type    | Source                                                                              |
+| ---------------| ------------------------------------------------------------------------------------|
+| Kubernetes     | [AppShield](https://github.com/aquasecurity/appshield/tree/master/kubernetes)       |
+| Dockerfile     | [AppShield](https://github.com/aquasecurity/appshield/tree/master/docker)           |
+| Terraform      | [tfsec](https://tfsec.dev/docs/aws/home/)                                           |
+
+CloudFormation and Ansible are coming soon.
```

Comment:
please add "for suggestions or issues regarding policy content, please open an issue under "Appshield" or "tfsec" repository.

Reply:
done https://github.com/aquasecurity/trivy/pull/1101/commits/bbaf804473891b843c92efbf969d3261279db0cd